java/ql/src/experimental/Security/CWE/CWE-489/ServletMain.qhelp

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
```

Comment:
I think the name of the `.qhelp` files need to match the check name. Could you please remove it, and add a dedicated ones for the two checks?

Reply:
I've created two separate <code>.qhelp</code> files for these two checks.